fvm/state/state.go

```diff
@@ -13,9 +13,9 @@ import (
 // TODO we started with high numbers here and we might
 // tune (reduce) them when we have more data
 const (
-	DefaultMaxKeySize         = 16_000        // ~16KB
-	DefaultMaxValueSize       = 256_000_000   // ~256MB
-	DefaultMaxInteractionSize = 2_000_000_000 // ~2GB
+	DefaultMaxKeySize         = 16_000      // ~16KB
+	DefaultMaxValueSize       = 256_000_000 // ~256MB
+	DefaultMaxInteractionSize = 20_000_000  // ~20MB
```

Comment:
We're happy with this size, going into master? do we want to adjust it at all?

Reply:
yeah, should be good to go, I also included the fixes to the error handling and bootstraping